doc/blog/Wasm-JS series/Blog - WASM.md

```diff
@@ -0,0 +1,672 @@
+
+   It's basically a control, having 2 properties, one code `Code` and another one for `Language`.
+
+2. Change the `MainPage.xaml` file to the following content:
+
+   ``` xaml
+   <Page
+       x:Class="PrismJsDemo.MainPage"
+       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+       xmlns:local="using:PrismJsDemo"
+       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+       mc:Ignorable="d">
+   
+       <Grid Padding="10">
+           <Grid.RowDefinitions>
+               <RowDefinition Height="Auto" />
+               <RowDefinition Height="*" />
+               <RowDefinition Height="*" />
+           </Grid.RowDefinitions>
+   
+           <TextBox x:Name="lang" Text="csharp" Grid.Row="0" />
+           <TextBox x:Name="code" Text="var x = 3;&#xA;var y = 4;" AcceptsReturn="True" VerticalAlignment="Stretch" Grid.Row="1" />
+   
+           <Border BorderBrush="Blue" BorderThickness="2" Background="LightBlue" Padding="10" Grid.Row="2">
+               <local:PrismJsView Code="{Binding Text, ElementName=code}" Language="{Binding Text, ElementName=lang}"/>
+           </Border>
+       </Grid>
+   </Page>
+   ```
+   
+3. Press CTRL-F5.  You should see this:
+   ![image-20200414144707425](image-20200414144707425.png)
+
+4. Press F12 (on Chrome, may vary on other browsers).
+
+5. Click on the first button and select the light-blue part in the app.
+   ![image-20200325132528604](image-20200325132528604.png)
+
+6. It will bring the DOM explorer to a `xamltype=Windows.UI.Xaml.Controls.Border` node. The `PrismJsView` should be right below after opening it.
+   
+   ![Html Explorer](image-20200325132859849.png)
+   The `xamltype="PrismJsDemo.Shared.PrismJsView"`) control is there!
+
+📝 The project is now ready to integrate PrismJS.
+
+## 3. Add JavaScript & CSS files
+
+🎯 In this section, PrismJS files are download from its website and put them as assets in the app.
+
+1. Go on this link: https://prismjs.com/download.html
+
+2. Choose desired Themes & Languages (`Default` theme + all languages is used for the demo)
+
+3. Press the `DOWNLOAD JS` button and put the `prism.js` file in the `WasmScripts` folder of the `.Wasm` project.
+
+   > Putting the `.js` file in this folder will instruct _Uno Wasm Bootstrap_ to automatically load the javascript file during startup.
+
+4. Press the `DOWNLOAD CSS` button and put the `prism.css` file in the `WasmCSS` folder of the `.Wasm` project.
+
+   > Putting the `.css` file in this folder will instruct _Uno Wasm Bootstrap_ to automatically inject a `<link>` html instruction in the resulting `index.html` file to load it by the browser.
+
+5. Right-click on the `.Wasm` project node in the solution explorer, and pick `Edit Project File` (it can also work by just selecting the project, if the `Preview Selected Item` option is activated).
```

Comment:
```suggestion
5. Right-click on the `.Wasm` project node in the Solution Explorer, and pick `Edit Project File` (it can also work by just selecting the project, if the `Preview Selected Item` option is activated).
```